prusti/src/driver.rs

```diff
@@ -175,7 +192,12 @@ fn main() {
         user::message(format!("Prusti version: {}", get_prusti_version_info()));
         info!("Prusti version: {}", get_prusti_version_info());
 
-        rustc_args.push("-Zpolonius".to_owned());
+        // TODO: decide if we want to keep this enabled; it means that we accept rust programs that
+        // rustc wouldn't. For example: prusti-tests/tests/verify/pass/nll-rfc/borrow_first.rs
+        if !config::no_verify() {
+            // Should no longer be needed since `get_body_with_borrowck_facts` doesn't require this
```

Comment:
When Vytatuas implemented `get_body_with_borrowck_facts` the `-Zpolonius` was needed. We can check if pthis is no longer true, but I don't expect so.